CHANGELOG.rst

```diff
@@ -14,6 +14,15 @@ Change Log
 Unreleased
 ~~~~~~~~~~
 
+[0.14.0] - 2017-01-15
```

Comment:
@mattdrayer thanks. No one knows for sure which PR will be merged first, so it makes little sense to try and predict what version should one's PR have